pegasus/sites.v3/code.org/views/display_language_lightbox.haml

```diff
@@ -37,7 +37,35 @@
       $.getJSON( "/v2/client-location", function( data ) {
         country_code = data['country_code'];
         if(country_code != 'US') {
-          $('#selectLanguage').modal('show')
+          $('#selectLanguage').modal('show');
+          $('#selectLanguage').on('shown.bs.modal', function () {
```

Comment:
I'd like to see this anonymous function refactored into a standalone function in this file for a few reasons:
1. Reduce the size of this `$( document ).ready` function thus making it more easy to understand.
1. Make it easier to tell where the focus trap logic begins and ends.
1. Make it easier for potential reuse in the future if we add other dialogs in non-React code.

Maybe the function would look something like
```
function focusTrap(modal_id) {
  return () => {
    var focusableEls = $(modal_id).find('a[href], button, ...
    ...
  };
}
```
You would call it like this:
```
$('#selectLanguage').on('shown.bs.modal', focusTrap('#selectLanguage'));
```